app/src/main/java/protect/card_locker/ManageGroupsActivity.java

```diff
@@ -187,7 +187,7 @@ private void moveGroup(View view, boolean up) {
         List<Group> groups = DBHelper.getGroups(mDatabase);
         final String groupName = getGroupName(view);
 
-        int currentIndex = DBHelper.getGroup(mDatabase, groupName).order;
+        int currentIndex = DBHelper.getGroupByName(mDatabase, groupName).order;
```

Comment:
Makes more sense to me if this uses the ID because you're not doing anything with the name and the ID should be the canonical way to refer to a group.

Reply:
I don't know how to handle this, as the only info that can be extracted from the view is the group name...